lxqtscreensaver.cpp

```diff
@@ -191,6 +191,8 @@ void ScreenSaverPrivate::_l_lockProcess_errorOccurred(QProcess::ProcessError)
 
 bool ScreenSaverPrivate::isScreenSaverLocked()
 {
+  if (QGuiApplication::platformName() == QStringLiteral("xcb"))
+  {
     XScreenSaverInfo *info = nullptr;
```

Comment:
Since the indentation is equal to 4 spaces in this source, you could either add enough spaces to respect the code style or just use this at the start instead:

```c++
    if (QGuiApplication::platformName() != QStringLiteral("xcb"))
    {
         return false;
    }
```

Reply:
Less lines the better, thanks!

Reply:
Since we use X11 functions below this, comparing with "xcb" would make more sense, as suggested above. "wayland" can be used where there are Wayland-specific codes.

BTW, you added a redundant empty line and forgot to indent the `if`.

Reply:
Done hopefully.  I thought it would be more explicit about wayland.

Reply:
It seems good now.

But I still think we shouldn't introduce `QGuiApplication` into `LXQt::ScreenSaver` but in the apps which use it. My reason is that `LXQt::ScreenSaver` may not be used in a GUI app, regardless of how *we* use it.

@palinek, is `QGuiApplication` safe here?

Reply:
By this usage, it should be perfectly safe as:
```  QString QGuiApplication::platformName()
  {
      return QGuiApplicationPrivate::platform_name ?
             *QGuiApplicationPrivate::platform_name : QString();
  } 
```
and the `QGuiApplicationPrivate::platform_name` is a static pointer to `QString`.